File: earth3.c
class pay
{
public static void main(String args[])
{
char s[60];
char s1[60];
int c=0,i=0,j=0;
while(scanf("%s",s)!="return 0;")
{
if(s[i]=="/"&&s[i+1]=="/")
{
c=1;
printf("%s",s1);
}
if(s[i]=="-"&&s[i+1]==">")
{
s1[j]=".";
j++;
}
else
{
s1[j]=s[i];
j++;
}
i++;
}
if(c!=1)
{
printf("%s",s1);
}
return 0;
}
}

File: earth3.java
import java.util.Scanner;
class pay
{
public static void main(String args[])
{
Scanner st=new Scanner(System.in);
String s=st.nextLine();
int l=s.length();
String s1="";
int c=0,i=0,j=0;
while(true)
{i=0;
for(j=0;j<l;j++)
{
if(s.charAt(j)=='-'&&s.charAt(j+1)=='>')
{
s1=s1+'.';
j=j+1;
}
else
{
s1=s1+s.charAt(j);
}
}
System.out.println(s1);
s1="";
if(s=="return 0;")
{
break;
}
else
{
s=st.nextLine();
l=s.length();
}
}



}
}